### ISSUE TITLE
ConTeXT writer: add \CSLBlock

### ISSUE THREAD

**Issue by denismaier** [jgm/pandoc#7361]
Unlike the latex writer the context writer doesn't wrap CSL elements with attribute `display=block` in a command (\CSLblock or so). This should probably be added.

**Comment by jgm** [comment]
Yeah, at this point we don't yet support CSL blocks on all writers.  Adding this should be similar to what we have in the LaTeX writer.

**Comment by denismaier** [comment]
I'll have a look. Maybe I can figure it out.
